Dart/thirdPartySrc/analysisServer/org/dartlang/analysis/server/protocol/AnalysisError.java

```diff
@@ -122,9 +121,10 @@ public static AnalysisError fromJson(JsonObject jsonObject) {
     Location location = Location.fromJson(jsonObject.get("location").getAsJsonObject());
     String message = jsonObject.get("message").getAsString();
     String correction = jsonObject.get("correction") == null ? null : jsonObject.get("correction").getAsString();
-    String code = jsonObject.get("code") == null ? null : jsonObject.get("code").getAsString();
+    String code = jsonObject.get("code").getAsString();
```

Comment:
I've reverted this line back to
```
String code = jsonObject.get("code") == null ? null : jsonObject.get("code").getAsString();
```
Otherwize the Dart plugin turns into a brick for older SDKs (and we are still compatible with SDK 1.12+)